### ISSUE TITLE
[BUG] 'color' command not working

### ISSUE THREAD

**Issue by leo-sar** [hedyorg/hedy#4752]
In level 4, the `color` command is introduced. However, it does not work.

level 4, drawing exercise:
color white
forward -80
color green
forward 50
color yellow
forward 50
color red
forward 50

<img width="471" alt="image" src="https://github.com/hedyorg/hedy/assets/150842805/7fb5a070-23db-41b9-8cde-2c36e1cadf04">

I am using Chrome on a Windows computer.

**Comment by jpelay** [comment]
Thanks for pointing this out! I'll look into this.

**Comment by jpelay** [comment]
Oops seems like @Felienne is already working on this in #4751

**Comment by Felienne** [comment]
Yes, I just heard from a teacher, will fix and deploy this tomorrow!
